Replicator/IncomingRev.cc

```diff
@@ -119,7 +119,66 @@ namespace litecore { namespace repl {
         if (_revMessage->noReply())
             _revMessage = nullptr;
 
-        _mayContainBlobs = jsonBody.containsBytes("\"digest\""_sl);
+        auto checkBlob = [](bool isDelta, alloc_slice jsonBody) -> bool {
```

Comment:
What about just `jsonBody.containsBytes("\"digest\""_sl)) || jsonBody.containsBytes("\"_attachments\""_sl))`?

Reply:
Well, this may give incorrect true. In theory, _mayContainBlobs being true is always right, because it says MAY, and the consequence is applying the delta earlier than necessary. In the other day when we were solving the other cbse that deleted attachment by db compact, I suggested to treat it alwasy true. In that case, we had delta _attachments actually with no changes.